media-libs/agar/agar-1.5.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A modern open-source, cross-platform toolkit for graphical applications"
+DESCRIPTION="Cross-platform GUI toolkit.  Agar provides a base framework and \
+a collection of GUI widgets from which GUI applications can be built.  Agar \
+can be styled and themed using a CSS-style engine."
+HOMEPAGE="http://libagar.org/index.html.en"
+COPYRIGHT="2002-2016 Hypertrition, Inc."
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="http://stable.hypertriton.com/agar/agar-1.5.0.tar.gz"
+CHECKSUM_SHA256="82342ded342c578141984befe9318f3d376176e5f427ae3278f8985f26663c00"
+
+ARCHITECTURES="!x86_gcc2 !x86 !x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 !x86"
+
+PROVIDES="
+	cmd:agar
```

Comment:
Since agar needs libsndfile which does not build with gcc2, we can add this, between SECONDARY_ARCHITECTURES and PROVIDES.
(Borrowed from games-board/stockfish/stockfish-8.recipe)
```
if [ "$targetArchitecture" != x86_gcc2 ]; then
	commandSuffix=$secondaryArchSuffix
	commandBinDir=$binDir
else
	commandSuffix=
	commandBinDir=$prefix/bin
fi
```
Then you can replace cmd:agar by
```
	cmd:agar$commandSuffix
```